rtest/CMakeLists.txt

```
@@ -15,11 +15,15 @@ endif()
 message(STATUS "Building RTEST for ROS distribution: ${ROS_DISTRO_TARGET}")
 
 # Validate supported distributions
-set(SUPPORTED_DISTROS "jazzy" "kilted")
+set(SUPPORTED_DISTROS "jazzy" "kilted" "rolling")
 if(NOT ROS_DISTRO_TARGET IN_LIST SUPPORTED_DISTROS)
   message(FATAL_ERROR "Unsupported ROS distribution: ${ROS_DISTRO_TARGET}. Supported: ${SUPPORTED_DISTROS}")
 endif()
 
+if(ROS_DISTRO_TARGET STREQUAL "rolling")
+  message(WARNING "ROS 2 Rolling distribution detected. This is experimental support, we don't guarantee that rolling will work correctly.")
+endif()
```

```suggestion
if(ROS_DISTRO_TARGET STREQUAL "rolling")
  message(WARNING "ROS 2 Rolling distribution detected. This is experimental support, we don't guarantee that rolling will work correctly.")
  ROS_DISTRO_TARGET = "kilted"
endif()
```
That should do the trick right?

---

A common term is then: `community support` :smile:

---

I'm not sure about this solution, but fine let's go with it.